src/dashboard/appContext.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const AppContext = React.createContext(null);
```

Comment:
Would `{}` be a better default value?
```suggestion
const AppContext = React.createContext({});
```